### ISSUE TITLE
DAP errors with MultiSelect.Filterable

### ISSUE THREAD

**Issue by kglickman** [carbon-design-system/carbon#6570]
I am seeing this with
"carbon-components": "10.14.0",
"carbon-components-react": "7.14.0",

It is also reproducible with the Carbon React storybook.

![image](https://user-images.githubusercontent.com/30597411/88697380-7c93ec00-d0d2-11ea-8a4c-9513647ab79f.png)

**Comment by liulrlr** [comment]
@rjhenriquez Just wonder if any updates for the fix?

**Comment by xiepingp** [comment]
@rjhenriquez, I test the MultiSelect component in https://react.carbondesignsystem.com/iframe.html?id=multiselect--filterable using Accessibility Checker, the violations are still happening, would you please take a look?
![6570 - carbon](https://user-images.githubusercontent.com/63960906/95550705-ed914180-0a3b-11eb-94ec-7278f5d8411e.png)

**Comment by tw15egan** [comment]
@xiepingp seems like the react website is still on `7.20.0`. I think this fix is included in `7.21.0`

**Comment by xiepingp** [comment]
Thanks @tw15egan for the information 😄 !

**Comment by xiepingp** [comment]
Hi @tw15egan, I retest this issue on carbon-components-react v7.25.0, below 2 violations are still there:
-Initial DOM focus on the combobox is not set on the text input when the combobox receives focus.
-The <input> element using 'aria-activedescendant' set to "carbon-multiselect-example-item-0" does not have its 'tabindex' attribute value set to 0 or -1.
https://react.carbondesignsystem.com/iframe.html?id=multiselect--filterable
![carbon-6570](https://user-images.githubusercontent.com/63960906/102045743-bb71d680-3e14-11eb-8ed4-1a44a291f9aa.png)
 
Could you please reope the issue and take a look?
Thanks!

**Comment by tw15egan** [comment]
Seems like the a11y violation is only popping up when an item is selected

**Comment by tw15egan** [comment]
@dakahn Seems like these are related? 
https://github.com/carbon-design-system/carbon/issues/7331 
https://github.com/IBMa/equal-access/issues/282#issue-747729478

**Comment by tw15egan** [comment]
Closing this since the initial issue has been fixed. The other a11y issues will be tracked in https://github.com/carbon-design-system/carbon/issues/7331
